### ISSUE TITLE
support .scss  entrypoint

### ISSUE THREAD

**Issue by nyrf** [ElMassimo/vite_ruby#50]
**Is your feature request related to a problem? Please describe.**

Add `typography.scss` to page: `<%= vite_stylesheet_tag 'typography', media: 'print' %>`
```
app/frontend: sourceCodeDir
  ├── entrypoints: entrypointsDir
  │   # only Vite entry files here
  │   └── application.js
  │   └── typography.scss
  └── components:
  │   └── App.vue
  └── channels:
  │   │── index.js
  │   └── chat.js
  └── stylesheets:
  │   └── my_styles.css
  └── images:
      └── logo.svg
```

**Comment by ElMassimo** [comment]
Thanks for the feature request, I plan to add it in the next iteration.

In the meantime, you can use SCSS by importing an `.scss` file from any JS file that is processed by Vite.

For example:

```ruby
<%= vite_javascript_tag 'typography' %>
```

```js
// app/frontend/entrypoints/typography.js
import '~/stylesheets/typography.scss'
```

**Comment by nyrf** [comment]
Thanks, I know that way, but import `.scss` file from JS file would cause a FOUC, https://github.com/ElMassimo/vite_ruby/discussions/48, i'll use the method until next iteration, thanks for your great work.

**Comment by ElMassimo** [comment]
Have in mind FOUC will only happen in development, but not in production—links to the stylesheets will be added.

**Comment by nyrf** [comment]
> Have in mind FOUC will only happen in development—in production links to the stylesheets will be rendered.

Yes, if i want to add a css file in some pages, like this:

`users/edit.html.erb`

```erb
<%= vite_stylesheet_tag 'user_edit'  %>
<h1>User Edit</h1>
```
I must create a new js file and import the `user_edit.scss`  file, then  use it in `users/edit.html.erb`

```erb
<%= vite_javascript_tag 'user_edit'  %>
<h1>User Edit</h1>
```

or use `user_edit.css`.

**Comment by nyrf** [comment]
I found  when i visit a page and render 404, then press browser back button, the stylesheet file  position wound't be right.

```
   <%= csrf_meta_tags %>
   <%= csp_meta_tag %>
   <%= vite_client_tag %>
   <%= vite_javascript_tag 'application' %>
   <%= vite_stylesheet_tag 'users.css' %>
```
![cs1](https://user-images.githubusercontent.com/299532/114311103-fca69880-9b1f-11eb-9a02-e4bed99a57a4.gif)

**Comment by ElMassimo** [comment]
@nyrf Just released the `vite-plugin-ruby@2.0.0` npm package, and the `vite_ruby` `1.2.3` gem.

In combination, they should allow you to use:

```erb
<%= vite_stylesheet_tag 'typography.scss', media: 'print' %>
```

**Comment by nyrf** [comment]
T'hanks for your great work, i'll update.
